updatecli/updatecli.d/updatecli.yml

```diff
@@ -0,0 +1,81 @@
+# This small test makes sure that updatecli is working properly on a repo.
+# To test this:
+#   have "UPDATECLI_GITHUB_ACTOR" env set to your github username
+#   have "UPDATECLI_GITHUB_TOKEN" env set to your github token
+#   have the latest version of updatecli installed
+#   'updatecli diff -v updatecli/values.yaml -c updatecli/updatecli.d/updatecli.yml'
+# In the future, more useful files should be added to this directory.
+---
+name: "Introduce updatecli to repo and validate basic functionality"
+# Make sure we can pull in github repos from multiple orgs
+scms:
+  rke2:
+    kind: "github"
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      username: "{{ requiredEnv .github.username }}"
+      token: '{{ requiredEnv .github.token }}'
+      owner: rancher
+      repository: rke2
+      branch: master
```

Comment:
Wouldn't it make sense to make this values variables in `updatecli/values.yaml`? As more Updatecli automatons are added, this can simplify the workflow files.

Reply:
I had that at first, but I noticed some synchronization issues. Every time we add a new updatecli config there ended up being a centralized file to update, which causes merge conflicts. 
I also found that I had to switch back and forth a lot when updating the files to make sure that the branch was set properly and that I was working on the correct repo. I know this is less DRY, but I think it makes it easier to update and maintain. Maintainers only have to worry about the one updatecli config they are changing, rather than juggling that and the values file.

Repos rarely change their owner or default branch (if they did change often I could see the need to centralize it), and this reduces the total number of lines that needs to be maintained (since we still need all of these lines in the updatecli config). The data is less centralized, and may be copied several times, but since that data is pretty static I think the trade-off for reduced complexity and less chance of merge conflict is worth it.